main.py

```diff
@@ -670,6 +670,31 @@ def delete_link_metadata(self, request: Request) -> JSONResponse:
         self.notify_topology_update()
         return JSONResponse("Operation successful")
 
+    @rest('v3/links/{link_id}', methods=['DELETE'])
+    def delete_link(self, request: Request) -> JSONResponse:
+        """Delete a down link from topology. It won't work for up links."""
+        link_id = request.path_params["link_id"]
+        try:
+            link = self.links[link_id]
+        except KeyError:
+            raise HTTPException(404, detail="Link not found.")
+        if link.status != EntityStatus.DOWN:
```

Comment:
It's an interesting idea to further require a particular state to allow the hard deletion here, it can simplify the cases we need to handle on other NApps and their side effects/consequences, even though it could also be handled as a `kytos/topology.link_down` with reason="link deleted". 

That said, if we'll follow this I'll recommend that we only allow deletion if `link.status == EntityStatus.DISABLED`, that way we know for sure that, this link has already been disabled and the event published, so NApps who need to do any clean up of flows when a link is disabled, like coloring that you mentioned, can do so by handling `kytos/topology.link_down` with reason="link disabled", and similarly if `kytos/topology.link_up` with reason="link enabled" is received then it pushes flows again. Wdyt?

Reply:
Changed to `DISABLED` and now sending the event `kytos/topology.link_down` with reason="link disabled"

For when a link is discovered, `coloring` manages well to push a flow and update its `self.switches` neighbors.

Reply:
Changed to `DISABLED` and now sending the event `kytos/topology.link_down` with reason="link disabled"

For when a link is discovered, `coloring` manages well to push a flow and update its `self.switches` neighbors.